Core/EventHandlers.lua

```diff
@@ -2071,6 +2077,25 @@ end
 
 local IsQuestFlaggedCompleted = C_QuestLog.IsQuestFlaggedCompleted
 
+function Rarity:grabAura()
+	local rotrBuff = 407063 -- Rocks on the Rocks buff
```

Comment:
Didn't you mean to use the shared constant here? At least I assume that's the case since you added one for the aura.

Reply:
yeah but i couldnt get it to work, guess ive forgotten how its correctly implemented

Reply:
You can simply reference `CONSTANTS.AURAS.ROCKS_ON_THE_ROCKS` in this case.